### PR TITLE
C0103 fixes 5 - last c0103 changes

### PR DESCRIPTION
**Pull request by davalb**
This PR is dependent on #1111.

### CODE REVIEW

**Comment by davalb** [comment]
I did the rebase, but I need to go over the code once more and fix some things. I will post here when I think that it is ready...

File: features/test_util.py [hunk 1]
```diff
@@ -27,11 +27,11 @@ def node_uri(node_index):
 def set_store_description(i):
     ws_send(i, 'update_settings',
             {'settings':
-             {'storeDescription': storeDescription(i),
+             {'storeDescription': store_description(i),
               'nickname': nickname(i)}})
```
Comment (davalb) [inline features/test_util.py]: not sure, if I have to change the storeDescription-index in this dictionary.

**Comment by davalb** [comment]
Hi @Renelvon , the tests for this branch succeed and openbazaar works locally now. Could you have a look at the changes? Here is one line I am not sure, what I should do with it: https://github.com/OpenBazaar/OpenBazaar/pull/1112/files#diff-402c811ebc4bdde56349a49418f674eeL30
I don't really understand the relevant code and therefore I am not sure if I should change the index to make it match the altered method-name. Do you know how to proceed here?

**Comment by davalb** [comment]
Hi, so this PR is working locally. Tests and pylint run without errors.  It would be great if somebody a little more experienced could have a look at `features/test_util.py ` line 30. I am unsure if I should have changed the `'storeDescription'` index there. Thanks!

**Comment by Renelvon** [comment]
I will have a look at in the next couple of days.

**Comment by davalb** [comment]
Awesome! Btw, I booked a ticket to FOSDEM. Looking forward to meet you face to face.

**Comment by Renelvon** [comment]
Nice to meet you, too! Drop me an e-mail if you want to hang out after FOSDEM. :)

**Comment by davalb** [comment]
Sorry, i didnt answer earlier. My battery died. We went to an Indian restaurant Punjab at avenue Adolphe buyl 18. You want to join us?

**Comment by hoffmabc** [comment]
We just finished dinner. Any thoughts on where to go?

**Comment by davalb** [comment]
Hey Brian, we are at the flagey place in the bar "le pantin". It is a nice bar but very crowded. I can recommend this area here are many other bars as well, but I think I will head home soon. Sorry for the bad communication on my part, I had problems with my phone.

File: features/steps/CryptoTransportLayer.py [hunk 2]
```diff
@@ -52,17 +52,17 @@ def cb(msg):
 def step_impl(context, i, j):
     i = int(i)
     j = int(j)
-    iLayer = context.layers[i]
-    jLayer = context.layers[j]
+    i_layer = context.layers[i]
+    j_layer = context.layers[j]
 
-    assert (ip_address(j), port, jLayer.guid, nickname(j)) in iLayer.dht.knownNodes
+    assert (ip_address(j), PORT, j_layer.guid, nickname(j)) in i_layer.dht.knownNodes
```
Comment (gasull) [inline features/steps/CryptoTransportLayer.py]: Long line.  Maybe this is better?:
```Python
known_node = (ip_address(j), PORT, j_layer.guid, nickname(j)) 
assert known_node in i_layer.dht.knownNodes
```

**Comment by hoffmabc** [comment]
Will need to rebase this bad boy.

**Comment by davalb** [comment]
Will take care of this tomorrow

File: features/steps/CryptoTransportLayer.py (hunk 2 above)
Comment (davalb) [inline features/steps/CryptoTransportLayer.py]: Changed like you suggested

Comment (gasull) [inline features/steps/CryptoTransportLayer.py]: :+1:

**Comment by davalb** [comment]
Hi, I rebased this, but as you can see the there are still failing tests. I could use somebody's help here, because my test-suite is broken locally and I don't know how to fix it.

If I run the `make` command, I get a bunch of errors like this:
``./venv/lib/python2.7/site-packages/coverage/htmlfiles/jquery.isonscreen.js: line 12, col 60, Unnecessary semicolon.```

If I run a specific unit_test on the command line like this ```python -m unittest test_network_util.py```
I get errors like this: ```ImportError: No module named node```

Can somebody help me with this?

**Comment by hoffmabc** [comment]
I'm just seeing Bandit issues. I'm not sure what's causing it. @Renelvon ?

**Comment by gubatron** [comment]
wish some of these had been done in separate commits to cherry pick a few of them. I bet it'll be hard to rebase these now.

how you doing @davalb , I remember you from FOSDEM :)

**Comment by hoffmabc** [comment]
Now you know why I left it alone so far lol. Rebase will be a killer.

**Comment by gubatron** [comment]
@hoffmabc Brian, take a look at the questions on your fix for cleaning up stale close nodes.

**Comment by hoffmabc** [comment]
I will but I'm AFK. Get to it soon.

**Comment by gubatron** [comment]
I'm about to pass out, so no rush.

**Comment by davalb** [comment]
Ok, it wasn't my intention to duplicate all these commits. Not sure how that happend. I thought I was rebasing this commit as usual, I'll try to revert that.

**Comment by hoffmabc** [comment]
Thanks.

**Comment by davalb** [comment]
Hi @gubatron! Doing good, Hope you are well, too. I did split up this commit in five parts. This is the fifth and last part. After merging this PR we are fully c0103-compliant and can enable the corresponding pyling-flag.

I rebased this PR again. Maybe we can finish this PR now once and for all. I think this PR looks worse than it actually is. Most of the changes are changes of variable-names to make them compliant with the pylint-requirements.

**Comment by gubatron** [comment]
LGTM

**Comment by harrymmmm** [comment]
there's another reference to get_NAT_status in node/openbazaar.py

**Pull request merged**